renderer.js

```diff
@@ -1282,6 +1282,24 @@ class SDFGRenderer {
         }
     }
 
+    visible_elements() {
+        let curx = this.canvas_manager.mapPixelToCoordsX(0);
+        let cury = this.canvas_manager.mapPixelToCoordsY(0);
+        let endx = this.canvas_manager.mapPixelToCoordsX(this.canvas.width);
+        let endy = this.canvas_manager.mapPixelToCoordsY(this.canvas.height);
+        let curw = endx - curx;
+        let curh = endy - cury;
+        let elements_light = {
```

Comment:
light?

Reply:
A leftover; initially the function returned the full element objects, light was used to indicate that it's only the IDs and not the full elements.